ui/src/flux/components/TagListItem.tsx

```diff
@@ -101,7 +101,11 @@ export default class TagListItem extends PureComponent<Props, State> {
                   <LoadingSpinner style={this.spinnerStyle} />
                 )}
               </div>
-              {!!count && `${count} total`}
+              {!!count && (
+                <div className="flux-schema--count">{`${count} Tag Value${
+                  count > 1 ? 's' : ''
```

Comment:
In an effort to try an get rid of ternaries, would you consider using a getter to return 's' or '' here instead?

Reply:
Refactored the whole "label" into a getter and replaced ternaries